### PR TITLE
EDGE-607: update to remove from and digits from verify endpoint

### PR DESCRIPTION
**Pull request by psherk-bw**
### What
We are removing the from and digits fields from the `/verify` endpoint since they aren't being used for anything. See related PR [here](https://github.com/Bandwidth/mfa_api/pull/69)

On the API side, this will be backwards compatible with current users supplying from and digits. If they supply the now two unneeded body parameters, our API will just ignore them.

### Why
Remove redundant code and update docs

### CODE REVIEW

**Comment by carrington-bw** [comment]
Can you test hitting the verify endpoint in the various combinations of before and after, with and without these parameters, to make sure it's backwards compatible?

**Comment by psherk-bw** [comment]
Wait to merge until https://github.com/Bandwidth/mfa_api/pull/69 is merged and deployed in Prod

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bandwidth/mfa",
-  "version": "1.1.1",
+  "version": "1.2.0",
```
Comment (jmulford-bw) [inline package.json:3]: Can we change this to `2.0.0` since the SDK is having a breaking change with the removal of the required fields?

**Review by jmulford-bw — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (carrington-bw) [inline package.json:3]: I don't think we update the version until you're ready to publish. IOW, updating the version is a separate step that's done at publish time

Comment (psherk-bw) [reply package.json]: updated

**Review by jmulford-bw — APPROVED** [review]

**Comment by psherk-bw** [comment]
> Should this be updated to use `main`?

@hamermike good catch, it should. I'm opening up the github action changes as a different PR https://github.com/Bandwidth/node-mfa/pull/8

**Pull request merged**